website/docs/r/healthcare_medtech_service_fhir_destination.html.markdown

```diff
@@ -11,11 +11,64 @@ description: |-
 Manages a Healthcare Med Tech Service Fhir Destination.
 
 ```hcl
-resource "azurerm_healthcare_medtech_service_fhir_destination" "test" {
+resource "azurerm_resource_group" "example" {
+  name     = "example"
+  location = "West Europe"
+}
+
+data "azurerm_client_config" "current" {
+}
+
+resource "azurerm_healthcare_workspace" "example" {
+  name                = "example"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+resource "azurerm_eventhub_namespace" "example" {
+  name                = "tfexample-ehn"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  sku                 = "Standard"
+}
+
+resource "azurerm_eventhub" "example" {
+  name                = "tfexample-eh"
+  namespace_name      = azurerm_eventhub_namespace.example.name
+  resource_group_name = azurerm_resource_group.example.name
+  partition_count     = 1
+  message_retention   = 1
+}
+
+resource "azurerm_eventhub_consumer_group" "example" {
+  name                = "$default"
+  namespace_name      = azurerm_eventhub_namespace.example.name
+  eventhub_name       = azurerm_eventhub.example.name
+  resource_group_name = azurerm_resource_group.example.name
+}
+
+resource "azurerm_healthcare_medtech_service" "example" {
+  name         = "mt"
```

Comment:
we intentionally name resources `example-{thing}` in examples, can we update these?